package.json

```diff
@@ -3,6 +3,7 @@
   "version": "2.6.2-dev",
   "description": "Twilio's JavaScript Voice SDK",
   "main": "./es5/twilio.js",
+  "module": "./mjs/twilio.js",
   "types": "./es5/twilio.d.ts",
```

Comment:
When importing as an esm, does the application imports from es5 folder since the `type` as defined here in package.json points to es5 folder?

Reply:
I could not find a way to specify different directories for types for ESM and CommonJS. Please let me know if you find a way to do it.